tasks/src/image-to-image/about.md

```diff
@@ -1,3 +1,13 @@
+## Most Used Model for the Task
+
+Pix2Pix is a popular model used for image to image translation tasks. It is based on a conditional-GAN (generative adversarial network) where instead of a noise vector a 2D image is given as input. More information about Pix2Pix can be retreived from the below link where the associated paper and Github repository can be found.
+
+https://phillipi.github.io/pix2pix/
+
+Below images show some of the examples shared in the paper that can be obtained using Pix2Pix. There are various cases this model can be applied to. It may do relatively simple things like converting a grayscale image to its colored version. But more importantly it can generate realistic pictures from rough sketches (as in the purse example) or from painting like images (as in the street and facade examples below).
```

Comment:
```suggestion
Below images show some of the examples shared in the paper that can be obtained using Pix2Pix. There are various cases this model can be applied on. It is capable of relatively simpler things, e.g. converting a grayscale image to its colored version. But more importantly, it can generate realistic pictures from rough sketches (can be seen in the purse example) or from painting-like images (can be seen in the street and facade examples below).
```
simpler things compared to what?

Reply:
I meant 'simpler' compared to the tasks that follow the coloring image example (i.e. purse, street and facade examples). Since instead of small manipulations of pixel changes; these tasks contain generation of new details to input image such as lightning, texture, color gradients.